akka-docs-dev/rst/stages-overview.rst

```diff
@@ -0,0 +1,134 @@
+drop                   the specified number of elements has been dropped already                                                                   the specified number of elements has been dropped and downstream backpressures                                                  Upstream completes
+take                   the specified number of elements to take has not yet been reached                                                           downstream backpressures The specified number of elements has been emitted, or upstream completes                               The defined number of elements has been taken or upstream completes
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+
+Asynchronous processing stages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These stages encapsulate an asynchronous computation, properly handling backpressure while taking care of the asynchronous
+operation at the same time (usually handling the completion of a Future).
+
+**It is currently not possible to build custom asynchronous processing stages**
+
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+Stage                  Emits when                                                                                                                  Backpressures when                                                                                                              Completes when
+=====================  =========================================================================================================================   ==============================================================================================================================  =====================================================================================
+mapAsync               the Future returned by the provided function finishes for the next element in sequence                                      the distance between the first uncompleted Future and between the last processed Future is larger than the defined parallelism  Upstream completes [1]_
+mapAsyncUnordered      a Future returned by the provided function finishes for any of the pending Futures                                          downstream backpressures and the number of pending Futures equals to the defined parallelism                                    Upstream completes [1]_
```

Comment:
the condition is not the same. This is unordered, so this can get rid of Futures as soon as they finish and downstream accepts. The ordered version can only get rid of the Future in the front slot.